test/indexTest.js

```diff
@@ -1,23 +1,112 @@
+describe("index.js", () => {
+  describe("defines saturdayFun function declaration as specified", function() {
+    it("function exists", function() {
+      expect(saturdayFun).to.exist
+    })
 
+    it("uses the default activity 'roller-skate' when no arguments are passed", function() {
+      expect(saturdayFun()).to.equal("This Saturday, I want to roller-skate!")
+    })
 
-describe("index.html", () => {
-  it("includes 'hello'", () => {
-    expect(document.querySelector('body').innerHTML).to.include('hello')
+    it("permits the default activity to be overriden", function() {
+      expect(saturdayFun("bathe my dog")).to.equal("This Saturday, I want to bathe my dog!")
+    })
   })
 
-  it("includes div", () => {
-    expect(document.querySelector('body').innerHTML, "no empty div element found within body").to.include('<div></div>')
+  describe("defines mondayWork function expression as specified", function() {
+    it("function exists", function() {
+      expect(mondayWork).to.exist
+    })
+
+    it("uses the default activity 'go to the office' when no arguments are passed", function() {
+      expect(mondayWork()).to.equal("This Monday, I will go to the office.")
+    })
+
+    it("permits the default activity to be overriden", function() {
```

Comment:
just to be clear - this is no different from the previous test, right? Both are solved in similar fashion:

```
function saturdayFun( actvity = 'roller-skate' ) {
  return `This Saturday, I want to ${actvity}!`
}

function mondayWork( activity = 'go to the office' ) {
  return `This Monday, I will ${activity}.`
}
```

but they show up at different parts of the readme, suggesting they might involve different tasks

Reply:
One uses a function declaration; other fn expression. JS has no way to know the diff afaict.